.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,20 +1,5 @@
  <!-- If this is your first PR, welcome! Please make sure you read the [contributing guidelines](../CONTRIBUTING.md). -->
 
- <!-- Please label this pull request according to what type of issue you are addressing (see ../CONTRIBUTING.md) -->
```

Comment:
I would suggest changing this to add a note of what to put in a PR title to kick off the autolabeler, possibly with a checkbox item to make sure it's done. Some of the checkboxes we might also want to get rid of since they aren't always applicable